src/EnergyPlus/UnitarySystem.cc

```diff
@@ -11066,7 +11086,8 @@ namespace UnitarySystems {
 
         Real64 FanPartLoadRatio = PartLoadRatio;
         if (this->m_SimASHRAEModel) FanPartLoadRatio = this->FanPartLoadRatio;
-        int SpeedNum = max(this->m_CoolingSpeedNum, this->m_HeatingSpeedNum);
+        if (this->m_EconoPartLoadRatio > 0) FanPartLoadRatio = this->m_EconoPartLoadRatio;
+        int SpeedNum = max(this->m_CoolingSpeedNum, this->m_HeatingSpeedNum, this->m_EconoSpeedNum);
```

Comment:
So economizer fan speed gets top priority here. That makes sense. And I don't think SimASHRAEModel works with multispeed coils (I actually don't recall, it's been a while) so this seems OK. It's also good you understand what's happening here, the fan PLR and coil PLR are independent here. I wonder how this interacts with the DXCoil flow per ton limits warnings ?

Reply:
I did add some DXCoil flow per ton limit warning "bypass" when the `EconomizerFirst` option is used. I added a comment for that above: https://github.com/NREL/EnergyPlus/pull/9987#discussion_r1235521796.

Reply:
That's fine. The flow per ton warning is not useful in the real world. There is an issue on very low flow per ton regarding coil freeze. A high flow per ton doesn't do anything other than waste fan power. And coil freeze can be tested using the coil outlet temp. So again, those warnings are mostly useless.